docs/releases/v2020.2.rst

```diff
@@ -189,3 +190,13 @@ Known issues
   (`#94 <https://github.com/freifunk-gluon/gluon/issues/94>`_)
 
   Reducing the TX power in the Advanced Settings is recommended.
+
+* The MAC address of the WAN interface is modified even when Mesh-on-WAN is disabled
+  (`#496 <https://github.com/freifunk-gluon/gluon/issues/496>`_)
+
+  This may lead to issues in environments where a fixed MAC address is expected (like VMware when promiscuous mode is
+  disallowed).
```

Comment:
This is fixed when using vxlan

Reply:
i will add this to this PR and the other PRs in the evening